stream_alert/shared/lookup_tables/table.py

```diff
@@ -0,0 +1,56 @@
+from stream_alert.shared.logger import get_logger
+
+LOGGER = get_logger(__name__)
+
+
+class LookupTable(dict):
```

Comment:
if these are just embedding a driver that handles the logic, can you remind me again why we need to subclass `dict` here?

Reply:
No reason; forgot to remove the `dict` after our discussion